streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java

```diff
@@ -55,6 +67,16 @@
     private final StreamsBuilder builder = new StreamsBuilder();
     private final Properties props = StreamsTestUtils.topologyTestConfig(Serdes.String(), Serdes.String());
 
+    private final Consumed<Windowed<Long>, String> windowConsumed = Consumed.with(WindowedSerdes.timeWindowedSerdeFrom(Long.class), Serdes.String());
+    private MaterializedInternal<Long, String, WindowStore<Bytes, byte[]>> materializedForWindow;
+    private InternalNameProvider nameProvider;
+
+    @Before
+    public void setUp() {
+        WindowBytesStoreSupplier supplier = Stores.persistentWindowStore("store", 10L, 3, 5L, false);
```

Comment:
As mentioned in the KIP discussion thread: it seem unnecessary complex for user, to specify all those value. The only parameter that is mandatory is the window size. If people call `KGroupedStream#windowBy` all other parameters also optional; they should be optional when reading a topic, too.